// -*-c++-*- VCity project, 3DUSE, Liris, 2013, 2014
////////////////////////////////////////////////////////////////////////////////
#ifndef DIALOGSTATE_H
#define DIALOGSTATE_H
////////////////////////////////////////////////////////////////////////////////
#include <QDialog>
#include "libcitygml/URI.hpp"

////////////////////////////////////////////////////////////////////////////////
namespace Ui {
class DialogState;
}
////////////////////////////////////////////////////////////////////////////////
class DialogState : public QDialog
{
    Q_OBJECT

public:
    explicit DialogState(QWidget *parent = 0);
    ~DialogState();

    void addState(const vcity::URI& uri);

private slots:
    void addFeatureBox();

private:
    Ui::DialogState *ui;
};
////////////////////////////////////////////////////////////////////////////////
#endif // DIALOGSTATE_H
